src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -558,4 +558,8 @@ describe('Test suite for mappings on existing concepts', () => {
     instance.removeAnswerRow('uniqueKey', true, 'url', 'name', 'code', true);
     expect(spy).toHaveBeenCalled();
   });
+  it('should call removeAnswer and remove the anwser row when a user clicks the remove button while creating a Q-A concept', () => {
```

Comment:
Can you point me to the file and line which has the removeAnswer function?

Reply:
`src/components/dictionaryConcepts/containers/EditConcept.jsx` on line 450

Reply:
Is that where the function is called or defined?

Reply:
In the same file, move to line 588 from which `removeSelectedAnswer` can be found `src/redux/actions/concepts/dictionaryConcepts.js`

Reply:
If you look at the line 588 in the same file, you can see the relationship with `removeSelectedAnswer`. `removeSelectedAnswer` is defined in `src/redux/actions/concepts/dictionaryConcepts.js` on line 215. It is then called in the file as shown in my previous comment.

Reply:
Are you talking about removeSelectedAnswer  or removeAnswer?

Reply:
`removeSelectedAnswer` is imported into `src/components/dictionaryConcepts/containers/EditConcept.jsx`. It is then mapped to `removeAnswer` and called on line `450` of in `EditConcept.jsx`